<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ODEClass</class>
 <widget class="QMainWindow" name="ODEClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ODE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>291</width>
      <height>541</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QDoubleSpinBox" name="ballVxBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <double>360.990000000000009</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="ballVyBox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>62</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <double>30.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Gun Barrel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Azimuth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>110</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Elevation</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>310</y>
        <width>211</width>
        <height>121</height>
       </rect>
      </property>
      <property name="title">
       <string>Integral Method</string>
      </property>
      <widget class="QRadioButton" name="ballEularRadio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>119</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Eular</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="ballMidPointRadio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>119</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mid Point</string>
       </property>
      </widget>
     </widget>
     <widget class="QSpinBox" name="gunHeight">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>60</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="ballMassBox">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>200</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="maximum">
       <number>9999</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Projectile</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Mass</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Force</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="ballPowerAmountMass">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>260</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>260</y>
        <width>121</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Powder Amount</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ballStopBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ballStartBtn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>450</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QSpinBox" name="springStiffBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>15</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="SpringMassBox">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>110</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Spring</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>91</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Stiffness k:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Mass m:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>kg/s^2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>110</y>
        <width>66</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>kg</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>221</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string>Intergration Method</string>
      </property>
      <widget class="QRadioButton" name="springEularRadio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>119</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Eular</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="springMidPointRadio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>119</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mid Point</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="springStartBtn">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>390</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="springStopBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1129</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ode.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ballStartBtn</sender>
   <signal>clicked()</signal>
   <receiver>ODEClass</receiver>
   <slot>BallStartPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>995</x>
     <y>630</y>
    </hint>
    <hint type="destinationlabel">
     <x>962</x>
     <y>660</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ballStopBtn</sender>
   <signal>clicked()</signal>
   <receiver>ODEClass</receiver>
   <slot>BallStopPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>850</x>
     <y>632</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>springStartBtn</sender>
   <signal>clicked()</signal>
   <receiver>ODEClass</receiver>
   <slot>SpringStartPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1026</x>
     <y>537</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>629</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>springStopBtn</sender>
   <signal>clicked()</signal>
   <receiver>ODEClass</receiver>
   <slot>SpringStopPlay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>881</x>
     <y>524</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>586</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BallStartPlay()</slot>
  <slot>BallStopPlay()</slot>
  <slot>SpringStartPlay()</slot>
  <slot>SpringStopPlay()</slot>
 </slots>
</ui>
